template-tarot.js

```diff
@@ -70,9 +97,9 @@ var TarotCard = (function(){ //<-- Superclass is parameter, but equals Card
 
 	// Override some other superclass methods and resources:
 
-	var suitNames = ["Cups","Pentacles","Swords","Wands"];
+	var suitNames = ['',"Cups","Pentacles","Swords","Wands"];
```

Comment:
This blank item is causing your suitNames to cycle through 5 items instead of 4, with one of them being blank.